assets/js/googlesitekit/modules/index.js

```diff
@@ -21,10 +21,20 @@
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
 import { createModuleStore } from './create-module-store';
+// This import has a side-effect: it automatically registers the "core/modules"
+// store on `googlesitekit.data`.
+import { STORE_NAME } from '../datastore/modules';
```

Comment:
In #1298 we decided to have the widgets datastore in `assets/js/googlesitekit/widgets/datastore` (instead of `assets/js/googlesitekit/datastore/widgets`) - here it is the other way around. We should decide on one approach and stick with it. From a module standpoint the first makes more sense - it clarifies that the datastore is part of the `widgets` (here `modules`) module. The second approach makes more sense from a "let's keep all datastores together" standpoint - however we'll not be doing this for module stores at least anyway, so I'm still not fully convinced. Let's discuss further as needed.

Reply:
Fair enough. I'll move these into `assets/js/googlesitekit/modules/datastore` 👍